pom.xml

```diff
@@ -26,7 +26,7 @@
     <maven.deploy.skip>true</maven.deploy.skip>
 
     <gpg.skip>true</gpg.skip>
-    <spotbugs.version>4.0.0-RC1</spotbugs.version>
+    <spotbugs.version>4.6.0</spotbugs.version>
     <spotbugs.fail>true</spotbugs.fail>
```

Comment:
@mukundansundar , @artursouza : It looks like we have enabled spotbugs to fail on error and interrupt the build. Do we want it to block the build?  In newer version it has converted all the warnings to errors.

Reply:
If we upgrade, then we need to fix the spotbugs errors.